requirements.txt

```diff
@@ -85,18 +88,19 @@ pep8==1.7.0
 pep8-naming==0.4.1
 pexpect==4.2.1
 pickleshare==0.7.4
+Pillow==4.3.0
 positional==1.1.1
 prettytable==0.7.2
 prompt-toolkit==1.0.9
 prospector==0.12.4
 psycopg2==2.7.3.2
 ptyprocess==0.5.1
 pycodestyle==2.0.0
+pycryptodomex==3.7.0
 pydocstyle==1.0.0
 pyflakes==1.3.0
 Pygments==2.1.3
-# Pylint 1.6.* does not work with prospector 0.12.4, so we have to use 1.5.6 for now.
-# See: https://github.com/landscapeio/prospector/issues/183
```

Comment:
@giovannicimolin, why have you removed this comment? This needs to be present just above the line with the pylint requirement.

Reply:
I've ran pip freeze > requirements.txt, just fixed it by only adding the necessary requirements at the bottom of the file.